docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -86,8 +86,9 @@ The Kyma installation process requires installation data specified in the `insta
 - `__ADMIN_GROUP__` for the additional admin group. This value is optional.
 - `__ENABLE_ETCD_BACKUP_OPERATOR__` to enable or disable the `etcd` backup operator. Enter `true` or `false`.
 - `__ETCD_BACKUP_ABS_CONTAINER_NAME__` for the Azure Blob Storage name of `etcd` backups. You can leave the value blank when the backup operator is disabled.
+- `__ENABLE_ETCD_BACKUP_CRONJOB__` to enable CronJob which executes periodically the [Etcd Backup][etcd-backup-app] application.
 
->**NOTE:** As the `etcd` backup feature is in development, set `__ENABLE_ETCD_BACKUP_OPERATOR__` to `false` and leave `__ETCD_BACKUP_ABS_CONTAINER_NAME__` blank.
```

Comment:
You added a new flag, but this only complicates the solution and also can lead to having some bug, e.g. what is the sense of having **__ENABLE_ETCD_BACKUP_OPERATOR__** set to false and **__ENABLE_ETCD_BACKUP_CRONJOB__** set to true?

so as we discussed, please introduce such pattern:
```yaml
global:
  etcdBackup:
      enabled: true
      secretName: etcd-backup-abs-credentials
```
then remove the `etcd-operator.backupOperator.enabled` flag and in **etcd-operator** and **service-catalog/etcd** charts use such statement `{{- if .Values.global.etcdBackup.enabled }}` to decide if backup should be enabled